samples/matter/light_bulb/sysbuild.cmake

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(CONF_FILE)
+    get_filename_component(CONFIG_FILE_NAME ${CONF_FILE} NAME)
+endif()
+
+# Set the paritions configuration
+if(NOT "${CONFIG_FILE_NAME}" STREQUAL "prj_no_dfu.conf")
+    set(PM_STATIC_YML_FILE ${ZEPHYR_NRF_MODULE_DIR}/samples/matter/light_bulb/configuration/${BOARD}/pm_static_dfu.yml PARENT_SCOPE)
+endif()
```

Comment:
this might work fine on linux, but what happens on windows if someone specifies `PRJ_NO_DFU.CONF` as the filename ?

Reply:
I think it will not work, but isn't it expected behavior? The same is going to happen if I call the main application file `PRJ.CONF` instead of `prj.conf`. With the current build system it will fail as well. I mean, I think that generally our build system is case sensitive regarding file names.

Reply:
Note **windows** in the above, windows is case insensitive, so `PRJ.CONF` and `prj.conf` both work and give you the same file.

Reply:
Will ignore this for now, can be opened as an issue later if a customer raises it as a problem